cric-exchange/src/api/PlayerAPI.ts

```diff
@@ -0,0 +1,5 @@
+import { getPlayerById } from '../dbaccess/PlayerAccessor';
```

Comment:
More of a question than a comment. Why do we need this file/Pattern. Why cant the App call PlayerAccessor directly. Separation of concerns?

Reply:
@rohitnair987

Reply:
Short answer - yes.

Layering: frontend - api - db accessor - db

DB accessors: are black boxes that take in params and give out data. The api should not have to worry about whether it uses mongo, mysql, ddb, azure or whatever. And this way, the db accessors are free to migrate as well without any concern.

API: The frontend code should not have to worry about passing exact params, structure the params into request/response etc. It should simply call a method with raw params and get a response. Our future plan should be separating out the api into a different project and hosting it in the cloud. That way, it can be decoupled.

Reply:
@rohitnair987 shall we create an item to track this?